lib/utils/KalturaUtils.js

```diff
@@ -466,6 +466,19 @@ KalturaUtils = {
 		});
 
 		request.end();
+	},
+
```

Comment:
so add handling for null instead of throwing an exception, this is the original behavior of JSON.parse anyway